lib/flipper-sequel.rb

```diff
@@ -0,0 +1,3 @@
+ActiveSupport.on_load(:sequel) do
```

Comment:
This could lead to some errors for people who aren't using active support. active support is [not a requirement for flipper](https://github.com/jnunemaker/flipper/blob/19f8c1aba5d56689d21dbe073f76318b81db2884/flipper.gemspec) or for flipper-sequel, so we can't use it here. The only reason the active record adapter does is because activesupport is a dependency of active record, which is listed in flipper active record.

I would just remove everything but the vanilla require here.